tsconfig.json

```diff
@@ -16,7 +16,7 @@
         ],
         "sourceMap": true,
         "rootDir": ".",
-        "alwaysStrict": true,
+        "strict": true,
```

Comment:
It's surprising that "strict" is more strict than "alwaysStrict" :-)

Reply:
Yeah, it really seems like a "gcc -Wall" scenario haha.